localstack/services/stepfunctions/asl/component/state/state_execution/state_task/service/state_task_service_callback.py

```diff
@@ -140,6 +140,17 @@ def _from_error(self, env: Environment, ex: Exception) -> FailureEvent:
             return self._get_callback_outcome_failure_event(env=env, ex=ex)
         return super()._from_error(env=env, ex=ex)
 
+    def _eval_body(self, env: Environment) -> None:
+        # Generate a TaskToken uuid within the context object, if this task resources has a waitForTaskToken condition.
```

Comment:
nit: it might be helpful to include more references to AWS docs for better traceability. For example: https://docs.aws.amazon.com/step-functions/latest/dg/connect-to-resource.html#connect-wait-token